tensorflow/c/experimental/filesystem/plugins/gcs/gcs_filesystem_test.cc

```diff
@@ -93,38 +93,53 @@ class GCSFilesystemTest : public ::testing::Test {
 };
 std::string GCSFilesystemTest::tmp_dir_;
```

Comment:
I missed this in a previous review but this is a huge linter warning internally. Can you make a PR to remove it?

See https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables

We only allow global/static variables if they are trivially destructible. Otherwise there might be issues with regards to the order the destructors get called.

Let's make a separate PR where we fix this. Maybe have a function that returns a `std::string*`. See https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/c/experimental/filesystem/modular_filesystem_test.cc;l=1692-1705;drc=43ff3c1e94b6f38608257302c7de3b69bab8fef9

Reply:
Ok I will send you a PR tomorrow